path_login_test.go

```diff
@@ -758,6 +774,35 @@ func TestLoginSvcAcctAndNamespaceSplats(t *testing.T) {
 	}
 }
 
+func TestLoginSvcAcctNamespaceSelector(t *testing.T) {
+	config := defaultTestBackendConfig()
+	config.saName = "*"
+	config.saNamespace = "non-default"
+	config.saNamespaceSelector = matchLabelsKeyValue
+	b, storage := setupBackend(t, config)
+
+	// test successful login
+	data := map[string]interface{}{
+		"role": "plugin-test",
+		"jwt":  jwtGoodDataToken,
+	}
+
+	req := &logical.Request{
+		Operation: logical.UpdateOperation,
+		Path:      "login",
+		Storage:   storage,
+		Data:      data,
+		Connection: &logical.Connection{
+			RemoteAddr: "127.0.0.1",
+		},
+	}
+
+	resp, err := b.HandleRequest(context.Background(), req)
+	if err != nil || (resp != nil && resp.IsError()) {
+		t.Fatalf("err:%s resp:%#v\n", err, resp)
+	}
```

Comment:
Thanks for the tests! 

It looks like we never evaluate `resp`. It's not totally clear what the test is testing. I think maybe checking some on the resp would make the test a bit clearer, or adding a comment, or adding something to the name of the test.